app/util/api_version_handler.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+
+class APIVersionHandler(object):
+    # List of all versions of the API.
+    _versions = [
+        1,
+        2
+    ]
+
+    @classmethod
+    def resolve_version(cls, header: str, uri: str):
```

Comment:
Maybe even `accept_header_value` since header technically refers to the "Accept" part as well and this is just the value